arches/app/templates/views/components/geocoders/mapbox.htm

```diff
@@ -0,0 +1,49 @@
+{% load i18n %}
```

Comment:
I think that these two templates (`mapbox.htm` and `mapzen.htm`) are identical, no? I think we could just create a single `geocoder.htm` template that both of these components could use.